docarray/array/array_stacked.py

```diff
@@ -26,15 +26,23 @@
     from pydantic.fields import ModelField
 
     from docarray.proto import DocumentArrayStackedProto
-    from docarray.typing import TorchTensor
-    from docarray.typing.tensor.abstract_tensor import AbstractTensor
 
 
 try:
     from docarray.typing import TorchTensor
 except ImportError:
     TorchTensor = None  # type: ignore
 
+try:
+    import tensorflow as tf  # type: ignore
+
+    from docarray.typing import TensorFlowTensor
```

Comment:
for torch i moved this thing to a helper in `utils`, so this check only has to be done once globally. Can we do the same for tf?